sdk/identity/Azure.Identity/src/Credentials/DefaultAzureCredential.cs

```diff
@@ -15,6 +15,7 @@ namespace Azure.Identity
     /// Provides a default <see cref="TokenCredential"/> authentication flow for applications that will be deployed to Azure.  The following credential
     /// types if enabled will be tried, in order:
     /// <list type="bullet">
+    /// <item><description><see cref="AzureDeveloperCliCredential"/></description></item>
```

Comment:
Is it correct that this comes first in the credential list now?

Reply:
I was surprised by that too. I think `EnvironmentCredential` should still be first in line.